pkg/universe.dagger.io/x/benjamin.reigner@epitech.eu/gcp/gcloud.cue

```diff
@@ -0,0 +1,53 @@
+package gcp
+
+import (
+	"universe.dagger.io/alpine"
+	"universe.dagger.io/docker"
+)
+
+// The gcloud tool, it allows us to have a simple container with the gcloud tool and an authenticated user
+#GCloud: {
+	config:  #Config
+	version: string | *"380.0.0"
+	packages: [pkgName=string]: version: string | *""
+
+	_alpine: alpine.#Build & {
+		"packages": {
+			packages
+			bash: {}
+			python: version: "3"
+			jq: {}
+			curl: {}
+			gcompat: {}
+		}
+	}
+
+	docker.#Run & {
```

Comment:
Could be replaced by `bash.#Run`.
Also, it's easier to follow if you stick `env` just below `command` instead of the top of the definition.
Basically the first field should be `input`